### ISSUE TITLE
Is there a way to make Cabal print test cases in real time?

### ISSUE THREAD

**Issue by mrkkrp** [haskell/cabal#3205]
When I use stack locally with test suite (Hspec), it prints tests cases (thier names, contexts) in real time. Now I need to force this behavior on Travis CI because my tests suite began to take longer than 10 minutes and I end up with:

> No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.

Tried `--show-details=streaming` without success (also in combination with `--test-option=--format=progress`).

**Comment by mrkkrp** [comment]
Also tried with `ghc-options: -threaded`, no luck.

**Comment by 23Skidoo** [comment]
For completness, the same question on SO: http://stackoverflow.com/questions/35756153/is-there-a-way-to-make-cabal-print-test-cases-in-real-time
tl;dr: appears to be something Travis-related, a workaround is to use `travis_wait`.

**Comment by ttuegel** [comment]
> Tried --show-details=streaming without success

That is what this option is designed to do, so either it is not working or Travis is using an older version of Cabal.

**Comment by mrkkrp** [comment]
I installed 1.22 version of Cabal with help of `multighc-travis` and currently I'm sure it's not a Cabal problem, or not only Cabal problem, but something that has to do with Travis as well. You can close the issue if you think it does not belong here. `travis_wait` worked for me.
